src/ios/CDVCamera.m

```diff
@@ -59,6 +59,59 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 @implementation CDVPictureOptions
 
+/*
+ API:
+ Camera:
+ - CameraPopoverHandle:
```

Comment:
it's a bit hard to follow these options since there's no indentation.
for example, is 'getPicture' also part of 'CameraPopoverHandle' ?